apps/sms/locales/sms.ar.properties

```diff
@@ -56,8 +56,9 @@ recipient[two]      = مستلِمان
 recipient[few]      = {{n}} مستلمين
 recipient[many]     = {{n}} مستلم
 recipient[other]    = {{n}} مستلم
-attachmentSize      = {{n}} كيلو بايت
-attachmentSizeMB    = {{n}} ميغابايت
+attachmentSizeB     = {{n}} {{unit}}
+attachmentSizeKB    = {{n}} {{unit}}
+attachmentSizeMB    = {{n}} {{unit}}
```

Comment:
Let’s use *one* new string instead, e.g.:
```ini
attachmentSize2 = {{n}} {{unit}}
```
… and update the related JS part so `l10nId` always remain the same.